orangecontrib/bio/genialis/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .genapi import Genapi
```

Comment:
I would suggest to name this Python module resolwe (not genialis), since resolwe is an open-source project. I suppose the goal of this module is to abstract the connection to two major versions of Genialis' platforms (given that they have somewhat different APIs). The new platform is based on Resolwe and since the old one will someday become obsolete, we may wish to stick to the new name. 

If you decide to rename, also rename the files in docs and tests.